linkerd/proxy/client-policy/src/lib.rs

```diff
@@ -157,6 +157,60 @@ impl ClientPolicy {
             backends: BACKENDS.clone(),
         }
     }
+
+    pub fn synthesize_forward(
+        meta: &Arc<Meta>,
+        timeout: time::Duration,
+        queue: Queue,
+        addr: SocketAddr,
+        metadata: EndpointMetadata,
+    ) -> Self {
```

Comment:
This feels like configuration to me, so I'd rather keep this in `app` and not in the library.

Reply:
Specifically: I want to make sure that we centralize all of the logic that creates default routes that allow traffic. All of that code needs to go away, ultimately. It should be closer to where it's needed/used.

Reply:
would you prefer this in the `linkerd-app-outbound` crate, or should it go in a (new) `linkerd_app_core::client_policy` module (or similar)?

Reply:
i think it's fine for it to be where it was alongside the other synthesize function; and then re-export them both from outbound for now.